app/src/js/components/Result.js

```diff
@@ -41,9 +41,12 @@ export const Result = ({ viewData = {}, isFeature, isCityFeature }) => {
   const fireClickAnalytics = () => {
     let category = 'Search';
     let action = 'Result Click';
-    if (isCityFeature) {
+    if (isSubFeature) {
       category = 'Discover';
-      action = 'City Feature Click';
+      action = 'Sub Feature Click';
```

Comment:
rip this data being relevant spanning this release but keeping it "city" would bother me